src/applications/appeals/testing/containers/IntroductionPage.jsx

```diff
@@ -26,7 +26,11 @@ const IntroductionPage = ({ route }) => {
 
   return (
     <div className="schemaform-intro vads-u-margin-left--1">
-      <va-breadcrumbs label="Breadcrumbs" class="vads-u-margin-left--neg1">
+      <va-breadcrumbs
+        label="Breadcrumbs"
+        class="vads-u-margin-left--neg1"
+        uswds
+      >
         <a href="/decision-reviews">Decision review and appeals</a>
```

Comment:
@Mottie I realize this PR is old but I was doing an audit for existing `va-breadcrumbs` in vets-website and I noticed that this one is using the `uswds` prop with the old list item structure. This is one of the few v3 components that that requires a different structure in order to use the USWDS variation. For example:

```
<VaBreadcrumbs
  breadcrumbList={[
    {
      href: '#one',
      label: 'Level one'
    },
    {
      href: '#two',
      label: 'Level two'
    },
    {
      href: '#three',
      label: 'Level three'
    }
  ]}
  label="Breadcrumb"
  uswds
/>
```

Storybook reference: https://design.va.gov/storybook/?path=/docs/uswds-va-breadcrumbs--default

Reply:
Thanks! Fixed in [#26059](https://github.com/department-of-veterans-affairs/vets-website/pull/26059)